app/models/task.rb

```diff
@@ -56,14 +56,14 @@ class Task < ApplicationRecord
                        }.fetch(visibility, public_access)
                      }
   scope :created_before_days, ->(days) { where(created_at: days.to_i.days.ago.beginning_of_day..) if days.to_i.positive? }
-  scope :average_rating, lambda {
-    select('tasks.*, COALESCE(avg_rating, 0) AS average_rating')
+  scope :overall_rating, lambda {
```

Comment:
This scope is currently untested (but used in the advanced filter). Could you please add a (single) test?

Reply:
Since the scope does not seem to be used anywhere else I merged it with min_stars. I still added a test though.

Reply:
If I see this right, the merged scope `min_stars` is tested in `spec/models/task_spec.rb`. Hence, this conversation can be resolved, I assume.